### PR TITLE
Misc fixes go brr

### LINKED ISSUE #12: No tags
Bot can&#39;t add tags to issues yet

_Issue created by Ginger [here](https://chat.stackexchange.com/transcript/106764?m=63010700#63010700)_

### LINKED ISSUE #22: s-s-status UwU
!!/status tingly doesn&#39;t have a high enough chance of inserting stutters and emoticons; pretty easy to fix (just diddle with the parameters to the <code>uwuify</code> function)

_Issue created by Ginger [here](https://chat.stackexchange.com/transcript/106764?m=63100158#63100158)_

### LINKED ISSUE #28: Make Vyxal Version a Required Argument of Issues Opened in the Main Repo
Now that version 3 is picking up momentum, there&#39;ll need to be clear differentiation between what is a v2 issue and a v3 issue. At this stage, just the bot command should have it mandatory

_Issue created by lyxal [here](https://chat.stackexchange.com/transcript/106764?m=63135125#63135125)_

### LINKED ISSUE #29: Improve Good Bot Regex
I should be able to send &quot;very good bot!&quot; and still trigger a response of &quot;:3&quot;

_Issue created by Ginger [here](https://chat.stackexchange.com/transcript/106764?m=63135423#63135423)_

### LINKED ISSUE #31: Fake commands
Some of the things listed in the commands list aren&#39;t actually commands. Let&#39;s fix that.

_Issue created by Ginger [here](https://chat.stackexchange.com/transcript/106764?m=63135511#63135511)_

### PR DESCRIPTION
**Pull request by GingerIndustries**
Closes #31
Closes #29 
Closes #28 
Closes #22 
Closes #12 
(whew!)

### CODE REVIEW

**Comment by Lyxal** [comment]
Does this force a version label on the main repo?

**Review by ysthakur — DISMISSED** [review]
i didn't even read this but i assume ginger's got this

**Review by ysthakur — CHANGES_REQUESTED** [review]
You closed #32 so this is my revenge

<sub>I'll approve once the comment above is resolved</sub>

File: vyxalbot2/__init__.py [hunk 1]
```diff
@@ -402,14 +402,20 @@ async def runCommand(
                         ][:5]
                     ),
                 )
-            case "issue-open":
+            case "!issue-open":
                 try:
+                    repo = args['repo'] if args['repo'] else self.config['baseRepo']
+                    if args['labels']:
+                        validLabels = [item async for item in self.gh.getiter(f"/repos/{self.config['account']}/{repo}/labels", oauth_token=(await self.appToken(self.gh)).token)]
+                        if not all(label in validLabels for label in args['labels'].split(' ')):
```
Comment (Lyxal) [inline vyxalbot2/__init__.py:410]: Don't forget to make sure a version is applied in the tags

File: vyxalbot2/__init__.py [hunk 2]
```diff
@@ -387,7 +387,7 @@ async def runCommand(
                 await self.room.reply(
                     event.message_id, random.choice(self.messages["hugs"])
                 )
-            case "repo-list":
+            case "!repo-list":
```
Comment (Lyxal) [inline vyxalbot2/__init__.py:390]: why the `!`s?

File: vyxalbot2/commands.py [hunk 3]
```diff
@@ -38,7 +38,7 @@
     (r"((please|pls|plz) )?(make|let|have) velociraptors maul (?P<user>.+)",): "maul",
     (r"(.* |^)(su+s(sy)?|amon?g ?us|suspicious)( .*|$)",): "sus",
     (r"(.* |^)([Ww]ho|[Ww]hat) (did|done) (that|this|it).*", r".*whodunit"): "blame",
-    (r"good bot",): "good-bot",
+    (r"(very |super |ultra |extremely )*good bot!*",): "!good-bot",
```
Comment (Lyxal) [inline vyxalbot2/commands.py:41]:
```suggestion
    (r"(much |very |super |ultra |extremely )*(good|great|excellent|gaming) bot!*",): "!good-bot",
```

**Review by Lyxal — DISMISSED** [review]

File: vyxalbot2/__init__.py (hunk 2 above)
Comment (GingerIndustries) [reply vyxalbot2/__init__.py:390]: > why the `!`s?

That was my stsem for fixing #31, but now I realize it doesn't actually work :| Will have to fix.

**Pull request merged**